### DISCUSSION TITLE
Unlock the diverse GitHub primitives on GitHub Mobile

### DISCUSSION THREAD

**Discussion by Akash1134** [community/community#134186] [Mobile]
![An image showing an outline of a phone with a list of GitHub Copilot prompts](https://github.com/user-attachments/assets/20c4cc02-e5b2-4382-9a17-81cc3a8f2639)

Copilot Chat in GitHub Mobile just got smarter! 🧠  It now has improved contextual awareness within the app, allowing you to ask it about the specific file or repository you are currently viewing. For enterprise licenses, Copilot Chat can also provide information about your open issues, pull requests, and discussions. This enhancement makes it easier than ever to get the assistance you need, exactly when you need it, all on the go. ⚡ 

- Download or update GitHub Mobile today from the [Apple App Store](https://apps.apple.com/us/app/github/id1477376905) or [Google Play Store](https://play.google.com/store/apps/details?id=com.github.android) to get started.
- [Learn more about GitHub Mobile](https://github.com/mobile) 

Share your feedback in the comments below to help us improve. 🚀

**Comment by Akash1134** [comment]
🚀

**Comment by udoxyu** [comment]
![image](https://github.com/user-attachments/assets/409f2bd1-d60a-4d56-9843-45914793a122)

![image](https://github.com/user-attachments/assets/b637c0d7-be84-4baf-bcc2-30d67a96076d)
